.golangci.yml

```diff
@@ -19,7 +19,10 @@ linters-settings:
   stylecheck:
     # added additional checks for comments in Go. 
     # Refer https://staticcheck.io/docs/options#checks for details
-    checks: ["all", "-ST1000", "ST1020", "ST1021", "ST1022"]
+    checks: ["all", "-ST1000", "-ST1003", "-ST1016", "-ST1020", "-ST1021", "-ST1022"]
+    dot-import-whitelist:
+      - github.com/onsi/ginkgo
+      - github.com/onsi/gomega
```

Comment:
We are already restricting dot imports here: 

https://github.com/Azure/ARO-RP/blob/5668a316afae97b690871edac2c59eb8f84efacb/hack/validate-imports/validate-imports.go#L18-L26

Probably makes sense to drop one of the checks: either in golangci or in our hack script. I would drop hack script validation (unless there is some extra value over golangci?)

Reply:
I think that we should remove the one from hack.